src/core/service.c

```diff
@@ -2134,6 +2160,9 @@ static int service_serialize(Unit *u, FILE *f, FDSet *fds) {
 
         unit_serialize_item(u, f, "forbid-restart", yes_no(s->forbid_restart));
 
+        if (s->watchdog_override_usec)
```

Comment:
nitpick: only bools and ptrs we check with the degrade-to-bool feature of C, for everything numeric please use numeric checks. Or in this case: please write this as `if (s->watchdog_override_usec != 0)`